examples/custom-agent-configuration/main.tf

```diff
@@ -9,9 +9,9 @@ module "lacework_k8s_datacollector" {
       "capsmode" : "enabled"
     }
     "fim" : {
-      "runat": "23:50"
-      "mode": "enable"
+      "runat" : "23:50"
```

Comment:
Could you remove these changes from the PR?

Reply:
https://github.com/lacework/terraform-kubernetes-agent/pull/72

Merge that one, then I'll rebase or something.